### PR TITLE
fix(nav): lazy loads notifications dropdown

### PR DESCRIPTION
**Pull request by dzucconi**
Closes [DIA-858](https://artsyproduct.atlassian.net/browse/DIA-858)

Notifications require a pretty large network of code that isn't needed on page load and can be deferred until the notifications bell is clicked.

[DIA-858]: https://artsyproduct.atlassian.net/browse/DIA-858?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by dzucconi** [comment]
— how...?

File: src/Components/NavBar/Menus/NavBarNotifications.tsx [hunk 1]
```diff
@@ -1,34 +1,33 @@
 import { Box } from "@artsy/palette"
-import { Notifications } from "Components/Notifications/Notifications"
-import { useEffect } from "react"
-import { useScrollLock } from "Utils/Hooks/useScrollLock"
+import loadable from "@loadable/component"
+import { NotificationsDropdownPlaceholder } from "Components/Notifications/Notifications"
```
Comment (dzucconi) [inline src/Components/NavBar/Menus/NavBarNotifications.tsx:3]: Oh, duh I need to isolate the placeholder!

**Review by damassi — APPROVED** [review]
🙏

**Comment by dzucconi** [comment]
OK that's more like it:

**Pull request merged**